### DISCUSSION TITLE
Having two tabs in SEB

### DISCUSSION THREAD

**Discussion by fkm97** [SafeExamBrowser/seb-win-refactoring#1026] [Q&A]
Hi,

I was just wondering, is it possible to have two or more tabs with different URLs open in SEB? If so, how would it be done? I came across this forum post long ago: https://sourceforge.net/p/seb/discussion/844844/thread/595fc832/ but I am not seeing the instructions on how to set this up.

I am looking to set up a new version(which has a start URL already) and has a couple of additional tabs with these URLs:
- [https://www.desmos.com/](https://www.desmos.com/)
- [https://translate.google.ca/?sl=auto&tl=it&op=translate](https://translate.google.ca/?sl=auto&tl=it&op=translate)

**Comment by dbuechel** [comment]
You can do this by using multiple browser windows and defining their position and size adequately, see https://safeexambrowser.org/windows/win_usermanual_en.html#AppearancePane and https://safeexambrowser.org/windows/win_usermanual_en.html#BrowserPane for main resp. additional window preferences.

**Reply by fkm97** [reply]
Where is the option to enable these additional browser windows and setting these additional URLs. That is something I am not seeing, could you possibly let me know the exact steps on what to do in the configuration?

**Reply by dbuechel** [reply]
You'll need to place the links in your web application resp. LMS with the correct target attribute (`target="_blank"`) so that they are opened in a new browser window when a user clicks them. Alternatively, you can automatically open them via JavaScript upon page visit.

**Comment by GregCSSC** [comment]
Hello to all,
I have same problem , I would like to open 3 web site inside SEB (in Yellow)
I programmed in HTML these 3 links with "target="_blank" and added it to SEB start web page 
But each link open in same page I would like it open in 3 different tabs
![Capture3liens](https://github.com/user-attachments/assets/143de241-f76c-41d2-80d2-a9a93108c97e)
Thank's a lot and have an nice day

**Reply by dbuechel** [reply]
Make sure you set the link policy correctly: https://safeexambrowser.org/windows/win_usermanual_en.html#BrowserPane > "Links requesting to be opened in a new browser window..." > "open in new window".

**Comment by GregCSSC** [comment]
Thank's for the reply,
Yes I have good settings:
![image](https://github.com/user-attachments/assets/2b2caa22-e40a-4cc8-a2b3-089c5eeb5722)
May be I have wrong setting in UserInterface ? 
![image](https://github.com/user-attachments/assets/17ac283c-899d-489d-bc22-377e5cb5d356)
I need these three websites to open in three different windows or three tabs so the student can navigate from one to the other without having to reopen it each time.

**Reply by dbuechel** [reply]
Try "Use browser window" under "Browser view mode".

**Reply by GregCSSC** [reply]
Hello, I tried your configuration but each time it opens the link in the same window
![image](https://github.com/user-attachments/assets/2a577e06-ab48-4080-9fa3-81af71e22880)

**Reply by danschlet** [reply]
Are you sure it's the same window? SEB opens additional windows on top of each other in the same size, so unless you move the topmost, you won't see the ones underneath. You can use Alt+Tab to cycle through all open windows as well.

**Comment by GregCSSC** [comment]
Yes , it was ALT+Tab , now it's working ! Thank's a lot and have a nice day
![Capture3liens](https://github.com/user-attachments/assets/eb92f8c0-b7f4-4112-b020-9c016b8cf3a8)
